instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_tracer.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry'
+
+module OpenTelemetry
+  module Instrumentation
+    module GraphQL
+      module Tracers
+        # GraphQLTracer contains the OpenTelemetry tracer implementation compatible with
+        # the GraphQL tracer API
+        class GraphQLTracer < ::GraphQL::Tracing::PlatformTracing
+          self.platform_keys = {
+            'lex' => 'graphql.lex',
+            'parse' => 'graphql.parse',
+            'validate' => 'graphql.validate',
+            'analyze_query' => 'graphql.analyze_query',
+            'analyze_multiplex' => 'graphql.analyze_multiplex',
+            'execute_query' => 'graphql.execute_query',
+            'execute_query_lazy' => 'graphql.execute_query_lazy',
+            'execute_multiplex' => 'graphql.execute_multiplex'
+          }
+
+          def platform_trace(platform_key, key, data)
+            return yield if platform_key.nil?
+
+            tracer.in_span(platform_key || 'test', attributes: attributes_for(key, data)) do |_span|
```

Comment:
```suggestion
            tracer.in_span(platform_key, attributes: attributes_for(key, data)) do |_span|
```

Reply:
What do you need the `|| 'test'` for here? `platform_key` will never be `nil` (but I supposed it could be `false`).

Reply:
🤕 that was placeholder that was accidentally re-added during a rebase. Gone now.